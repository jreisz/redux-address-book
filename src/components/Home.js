import React from 'react'

const Home = (props) => (
	<h1>
		Welcome home!
	</h1>
)

export default Home;