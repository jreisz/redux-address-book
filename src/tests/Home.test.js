import React from 'react';
import ReactDOM from 'react-dom';
import Home from '../components/Home';

describe('Home', () => {
  it('renders container', () => {
    const wrapper = document.createElement('div');
    ReactDOM.render(<Home />, wrapper);
    expect(wrapper).toMatchSnapshot();
  });
});
