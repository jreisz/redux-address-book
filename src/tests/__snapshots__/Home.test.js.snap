// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders container 1`] = `
<div>
  <h1>
    Welcome home!
  </h1>
</div>
`;
